lib/iser.c

```diff
@@ -812,6 +816,32 @@ static int iser_create_iser_conn_res(struct iser_conn *iser_conn) {
 	return ret;
 }
 
+/*
+ * iser_rdma_set_option() - set rdma options
+ *
+ * @cma_id:    connection manager id
+ * @iscsi:   iscsi context
+ */
+int iser_rdma_set_option(__attribute__((unused)) struct rdma_cm_id *cma_id, __attribute__((unused)) struct iscsi_context *iscsi) {
+#ifdef HAVE_RDMA_ACK_TIMEOUT
```

Comment:
Please rework this such that the #ifdef HAVE_RDMA_ACK_TIMEOUT occurs outside the iser_rdma_set_option() function and such that __attribute__((unused)) is only used for the function arguments if HAVE_RDMA_ACK_TIMEOUT  has not been defined.